### DISCUSSION TITLE
What is the last version of Rancher Desktop compatible with MacOS Big Sur ?

### DISCUSSION THREAD

**Discussion by nyue** [rancher-sandbox/rancher-desktop#6667] [Q&A]
I read the latest version requires Mac OS Catalina

**Comment by jandubois** [comment]
> I read the latest version requires Mac OS Catalina

Yes, macOS Catalina (10.15) is the minimum Version for Rancher Desktop on macOS. Big Sur is version 11, so is supported by Rancher Desktop as well.

We may stop supporting Catalina pretty soon, given that the OS is no longer supported by Apple, and GitHub Actions no longer provide runners for us to perform CI on it.

Note that Big Sur is also no longer supported by Apple (and GitHub), so we may need to drop support for it as well if we run into build problems for it.

But for now macOS 10.15 is still supported for Intel and macOS 11 for Apple M1+ chips (there never was ARM support in Catalina).

**Comment by simonboet** [comment]
@jandubois Hi, sorry to necro an old thread. But do you know if Catalina support has been ceased? The documentation for Rancher Desktop still says that Catalina is the minimum OS, but I downloaded the latest version and it states it is incompatible. If so, do you know which version would be the last working version? And additionally, would you recommend running it for personal use?

**Reply by jandubois** [reply]
> do you know if Catalina support has been ceased?

Unfortunately I don't. It is the bundled QEMU version that has issues.

I would try 1.15, and then move backwards from there, if that doesn't work.

> The documentation for Rancher Desktop still says that Catalina is the minimum OS

Thanks for pointing that out; we need to update the docs site.

> would you recommend running it for personal use?

I think running an older version will be fine. Just make sure you disable auto-update; otherwise you may get upgraded to a version that doesn't work.
